charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,8 +23,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 67.2.0
-appVersion: v0.79.0
+version: 67.3.0
```

Should I have just bumped the patch version here to correspond with the operator patch bump?

---

```suggestion
version: 67.2.1
```

(realized when re-reviewing this that I just bumped the minor by default but that might not be right)